designer/server/src/config.ts

```diff
@@ -71,6 +72,7 @@ const schema = joi.object<Config>({
   isDevelopment: joi.boolean().default(true),
   isTest: joi.boolean().default(false),
   sessionTtl: joi.number().required(),
+  fileDownloadPasswordTtl: joi.number().default(34560),
```

Comment:
Can you remove the default value and make it required please? Raise a PR for https://github.com/DEFRA/cdp-app-config/ with the values for each environment.

In that repo you can set it per environment or a global value. Use 2 days for dev, then the keep the other environments as per the ticket.